docs/src/main/paradox/google-cloud-pub-sub-grpc.md

```diff
@@ -38,8 +38,8 @@ The table below shows direct dependencies of this module and the second tab show
 
 @@@warning
 
-This connector contains code generated from Protobuf files which is bound to @extref:[Akka gRPC $akka-grpc.version$](akka-grpc:). This makes it @extref:[NOT binary-compatible](akka-grpc:/binary-compatibility.html) with later versions of Akka gRPC.
-You can not use a different version of Akka gRPC within the same JVM instance.
+This connector contains code generated from Protobuf files which is bound to @extref:[Apache Pekko gRPC $akka-grpc.version$](akka-grpc:). This makes it @extref:[NOT binary-compatible](akka-grpc:/binary-compatibility.html) with later versions of Apache Pekko gRPC.
```

Comment:
same here, maybe intended but references to `akka-grpc.version` etc.

Reply:
as for akka.version, I'd prefer to wait until we start using pekko-grpc in the builds to update this